reporting/terraform/main.tf

```diff
@@ -30,6 +31,26 @@ module "lambda_miro_inventory_transformer" {
     ES_DOC_TYPE = "miro_inventory_record"
   }
 
+  vhs_read_policy       = "${local.miro_inventory_vhs_read_policy}"
   trigger_topic_arn     = "${local.miro_inventory_topic_arn}"
   error_alarm_topic_arn = "${local.lambda_error_alarm_arn}"
 }
+
+module "lambda_sierra_transformer" {
+  source = "./reporting_lambda"
+
+  name        = "reporting_sierra_transformer"
+  description = "Transform sierra source data and send to ES."
+
+  environment_variables = {
+    ES_URL      = "${var.reporting_es_url}"
+    ES_USER     = "${var.reporting_es_user}"
+    ES_PASS     = "${var.reporting_es_pass}"
+    ES_INDEX    = "sierra"
```

Comment:
I think we should move this to a more specific index and include the creation data to avoid confusion and name clashes.